app/src/main/kotlin/com/wire/android/ui/common/topappbar/CommonTopAppBar.kt

```diff
@@ -0,0 +1,134 @@
+package com.wire.android.ui.common.topappbar
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Text
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.google.accompanist.systemuicontroller.rememberSystemUiController
+import com.wire.android.R
+import com.wire.android.ui.theme.wireColorScheme
+import com.wire.android.ui.theme.wireDimensions
+import com.wire.android.ui.theme.wireTypography
+
+@Composable
+fun CommonTopAppBar(commonTopAppBarViewModel: CommonTopAppBarViewModel) {
```

Comment:
Oh I totally missed it (although having seen it some times 🤦🏻 )

The thing is that for the created `CommonTopAppBar` I need to pass a ViewModel in order to be used inside it. Currently (this ticket) we will only set this to `HomeScreen` and `ConversationScreen`. 

Would it make sense keeping it like it is and handle its changes/content to `WireCenterAlignedTopAppBar` in the other [ticket where it will handle other screens](https://wearezeta.atlassian.net/browse/AR-2164)?

Reply:
Ok, but maybe we can already use `WireCenterAlignedTopAppBar` inside `CommonTopAppBar`? 😉